nd4j/nd4j-parameter-server-parent/nd4j-parameter-server-node/pom.xml

```diff
@@ -41,14 +41,34 @@
         </dependency>
         <dependency>
             <groupId>org.nd4j</groupId>
-            <artifactId>nd4j-parameter-server-status_2.10</artifactId>
+            <artifactId>nd4j-parameter-server-status_2.11</artifactId>
             <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>org.nd4j</groupId>
             <artifactId>nd4j-parameter-server</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>${logback.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>io.reactivex.rxjava2</groupId>
+            <artifactId>rxjava</artifactId>
+            <version>2.2.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.nd4j</groupId>
+            <artifactId>nd4j-native</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
```

Comment:
Can't have this dep here even under test scope, for CI CUDA-only builds:
https://github.com/deeplearning4j/deeplearning4j/blob/a948b1364329c7281aa2302524e2b9dcd6858cf2/nd4j/nd4j-parameter-server-parent/nd4j-parameter-server-node/pom.xml#L77